pkg/storaged/storage.scss

```diff
@@ -232,15 +232,17 @@ td.storage-action {
     overflow: visible !important;
 }
 
+.units-table {
+    width: 100%;
+}
```

Comment:
We could use PF utils instead.

https://www.patternfly.org/v4/utilities/sizing

The class `pf-u-w-100` will make something 100% width. It also supports breakpoints and other sizing, but this is a 1:1 change.

To use it: Just make sure the utils CSS is imported and add the class to the element.